### DISCUSSION TITLE
Is RTL8812cu supportted?

### DISCUSSION THREAD

**Discussion by Zjuheteng** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#292] [Q&A] [answered]
I have this question because my PC has a Realtek 8812CU Wireless LAN 802.11ac NIC for Wi-Fi and I wanted to know if this will work on the macOS Big Sur setup (Opencore)

**Comment by chris1111** [comment, marked as answer]
Not support in macOS only 8812AU is support
